cg/meta/workflow/mutant/quality_controller/quality_controller.py

```diff
@@ -0,0 +1,128 @@
+from pathlib import Path
+from cg.apps.lims.api import LimsAPI
+from cg.constants.constants import MutantQC
+from cg.meta.workflow.mutant.quality_controller.models import (
+    QualityMetrics,
+    SampleQualityResult,
+    CaseQualityResult,
+    QualityResult,
+)
+
+from cg.meta.workflow.mutant.quality_controller.result_logger import ResultLogger
+from cg.meta.workflow.mutant.quality_controller.utils import (
+    has_valid_total_reads,
+    internal_negative_control_qc_pass,
+    external_negative_control_qc_pass,
+    get_quality_metrics,
+    get_report_path,
+)
+from cg.meta.workflow.mutant.quality_controller.report_generator import ReportGenerator
+from cg.store.models import Case
+from cg.store.store import Store
+
+
+class QualityController:
+    def __init__(self, status_db: Store, lims: LimsAPI) -> None:
+        self.status_db = status_db
+        self.lims = lims
+
+    def quality_control(
```

Comment:
```suggestion
    def get_quality_control_results(
```

or a bit of a more meaningful name 🤔